src/ir/comp.rs

```diff
@@ -490,49 +490,67 @@ fn bitfields_to_allocation_units<E, I>(ctx: &BindgenContext,
     let mut unit_align = 0;
     let mut bitfields_in_unit = vec![];
 
+    // TODO(emilio): Determine this from attributes or pragma ms_struct
+    // directives. Also, perhaps we should check if the target is MSVC?
```

Comment:
We could, but I'd like to check with @upsuper in order to confirm that integration tests work fine for him with that bit.